source/MapDetailPanel.cpp

```diff
@@ -800,11 +800,16 @@ void MapDetailPanel::DrawInfo()
 			else
 			{
 				value -= localValue;
-				price += "(";
+				if(Preferences::Has(Preferences::PRICE_PARENS))
```

Comment:
Other preference checks just use the preference name directly. If you want to change it, open a separate refactor PR for all preferences.
```suggestion
				if(Preferences::Has("Show parenthesis on Job Board"))
```